config/params.php

```diff
@@ -102,8 +102,7 @@
             ],
         ],
         'annotated-entity-paths' => [
-            '@src/Entity',
-            '@src/Blog/Entity',
+            '@src',
```

Comment:
Is there a performance impact? Cycle needs to check all classes in that case

Reply:
Sure there is but that's done only once. Then schema is cached.

Reply:
Is it possible to warmup cache?

Reply:
Yes, with `./vendor/bin/yii cycle/schema/rebuild`.